packages/react/src/themes/teams/animations/scale.ts

```diff
@@ -3,47 +3,50 @@ import { easeOut, easeIn, easeEasy } from './timingFunctions'
 const scaleAnimations = {
   // Scale in w/ Fade -- Useful for opening modal dialogs
   scaleEnterFast: {
-    keyframe: {
+    keyframe: ({ delta }) => ({
```

Comment:
The changes looks good! Let's just extract the repeting functions in a reusable const, something like: 
```
const scaleKeyframe =  ({ delta }) => ({
  '0%': {	     
    transform: `scale(${delta})`,
    opacity: 0,
  },	     
  '100%': {	 
    transform: 'scale(1)',	   
    opacity: 1,	   
  },	     
}),
```

and then define the animations as:
```
scaleEnterFast: {
  keyframe: scaleKeyframe,
  duration: '500ms',
  timingFunction: easeOut,
  keyframeParams: { delta: '.9' },
  fillMode: 'forwards'
}
```